includes/functions/llms-functions-content.php

```diff
@@ -15,83 +15,131 @@
 	/**
 	 * Post Template Include
 	 *
-	 * Appends LLMS content above and below post content.
+	 * Adds LifterLMS template content before and after the post's default content.
 	 *
 	 * @since 1.0.0
 	 * @since 3.25.2 Unknown.
+	 * @since [version] TODO.
 	 *
 	 * @param string $content WP_Post post_content.
 	 * @return string
 	 */
 	function llms_get_post_content( $content ) {
 
 		global $post;
-		if ( ! $post instanceof WP_Post ) {
+		if ( ! $post instanceof WP_Post || ! in_array( $post->post_type, array( 'course', 'llms_membership', 'lesson', 'llms_quiz' ), true ) ) {
 			return $content;
```

Comment:
I'm just realizing the prior to this PR we basically passed all the posts' contents through the filter `llms_get_post_content`, while now we do that only for those "special" post types... mmm.
Am I wrong?
I think it's better this way, but this is a change in behavior... right?

Reply:
mmm yea, you're right...

It's both good and bad though...

I think the fact that we don't filter other content types is possibly a problem

The fact that we don't run them through `do_shortcode()` is beneficial

Reply:
Right?

Reply:
@eri-trabiccolo check my update please

Reply:
Yeah...

Reply:
@thomasplevy looks good, although I wonder why we have `do_shortcode` there.
Our filter is added at priority 10, while `do_shortcode` is added by wp core at priority 11.

Maybe it wasn't so back in the past?

Reply:
@eri-trabiccolo It's possible that it's never been needed. I'm not sure. I'm not digging into this right now.

Reply:
I'm totally with you